draft-ietf-ppm-dap.md

```diff
@@ -813,6 +800,31 @@ encoded value of the following form:
 "extension_type" indicates the type of extension, and "extension_data" contains
 information specific to the extension.
 
+### Upload Message Security
+
+The contents of each input share must be kept confidential from everyone but the
+client and the aggregator it is being sent to and clients must be able to
+authenticate the aggregator they upload to.
+
+HTTPS provides confidentiality between the client and the leader, but this is
+not sufficient since the helper's report shares are relayed through the leader.
+Confidentiality of report shares is achieved by encrypting each report share to
+a public key held by the respective aggregator. Clients fetch the public keys
+from each aggregator over HTTPS, allowing them to authenticate the server.
+
+Aggregators MAY require clients to authenticate when uploading reports. This is
+an effective mitigation against Sybil attacks in deployments where it is
+practical for each client to have an identity provisioned (e.g., a user logged
+into an online service or a hardware device programmed with an identity). If it
+is used, client authentication MUST use a scheme that meets the requirements in
+{{request-authentication}}.
+
+In some deployments, it will not be practical to require clients to authenticate
+(e.g., a widely distributed application that does not require its users to login
+to any service), so client authentication is not mandatory in DAP. In the
+absence of client authentication, implementations SHOULD deploy some mitigation
+against Sybil attacks.
```

Comment:
As it stands, this problem is unsolved. I don't think it's fair to nudge implementors until we have a better sense of what alternatives are possible. It might be worth pointing to an issue regarding Sybil attacks (we have at least one somewhere).

```suggestion
to any service), so client authentication is not mandatory in DAP.
```